packages/primitives/text-block/readme.md

```diff
@@ -0,0 +1,31 @@
+`Text-Block` is a fusion between `Block` and `Text`.
+
+It does not provide all properties each has to offer, it only provides the simplest API possible to deal with content in a `display: block` manner.
```

Comment:
I understand the "display: block" analogy, but technically it will not be so since we use flexbox 😅 

I guess just saying "to deal with text content as a block type element"